otherlibs/stdune/test/bytes_unit_tests.ml

```diff
@@ -31,23 +33,37 @@ let%expect_test _ =
     ; 12345678901L
     ; 123456789012L
     ; 1234567890123L
-    ]
-  in
-  List.iter ~f:(fun x -> Bytes_unit.pp x |> print_endline) bytes;
+    ; 12345678901234L
+    ; 123456789012345L
+    ; 1234567890123456L
+    ];
   [%expect
     {|
-      0B
-      1B
-      12B
-      123B
-      1.23kB
-      12.35kB
-      123.46kB
-      1.23MB
-      12.35MB
-      123.46MB
-      1.23GB
-      12.35GB
-      123.46GB
-      1.23TB |}]
+0 B
```

Comment:
Why is this feature changing serialisation (adding a space)?

Reply:
I added it to be more legible.

Reply:
Well, still, this PR keeps mixing different changes together which makes it hard to review and slows the review down. Why not stick to the point, which is adding new units, as the title says?

Reply:
You're right. I've split that change off now. Sorry about that.